swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/aws-lambda.md

```diff
@@ -1,193 +1,103 @@
 ---
 layout: ballerina-cloud-left-nav-pages-swanlake
 title: AWS Lambda
-description: Learn how to write and deploy AWS Lambda functions using ballerina
-keywords: ballerina, programming language, serverless, cloud, aws, lambda, cloud native
+description: AWS Lambda is an event-driven, serverless computing platform. The AWS Lambda extension provides the functionality to write AWS Lambda-compatible packages by exposing a Ballerina function as an AWS Lambda function.
```

Comment:
Do we need `AWS Lambda is an event-driven, serverless computing platform`?

Reply:
Updated.